docs/pages/blog/material-ui-is-now-mui.md

```diff
@@ -0,0 +1,99 @@
+card: true
+---
+
+Starting today, we are evolving our brand identity. We are clarifying the difference between our company and our products.
+
+- Material-UI: the organization is now called [**MUI**](https://github.com/mui-org/).
+- Material-UI: the MIT React library is now called [**MUI Core**](https://github.com/mui-org/material-ui).
+- Material-UI X: the set of advanced React components is now called [**MUI X**](https://github.com/mui-org/material-ui-x).
+
+The new identity matches with the increase scope of our focus.
+
+## The origin
+
+Material-UI started as a React implementation of the Material Design guidelines in 2014.
+The goal was simple, to empower the React developers to use Material Design.
+The community [loved it](https://news.ycombinator.com/item?id=8582439) and was craving for more, so we kept building.
+
+We have made it smooth for the community to contribute.
+We started actively reviewing pull requests and guiding issues.
+Things started to really take off with [the release of v1](/blog/material-ui-v1-is-out/) in 2018.
+The frontend community was actively migrating to React and in desperate need to move fast.
+
+Before we realized it, we had grown into the most popular React UI components library.
+We were heavily used and customized to build: customer-facing applications, internal tools, and mobile-hybrid applications.
+
+## A larger focus
+
+In our last survey, [60% fewer](/blog/2020-developer-survey-results/#comparison-with-last-year) developers were complaining about improving Material Design implementation than they did a year before.
+At the same time, 400% more developers were struggling to customize the components.
+
+It's the context in which we started working on v5 in 2019.
+Our primary focus was to revamp the **customization DX**.
+It had become clear that design and DX were key to unlock the next stage of growth.
+
+The release of this rebranding matches with the release of v5.0.0.
+It's a big deal.
+The new branding aligns with the increase of the company's area of focus.
+
+Our ultimate goal is to become the most effective and efficient tool to build UIs while making it accessible to the many.
+
+## Our new brand
+
+Let's dig in.
+
+### A new name
+
+We are breaking the strong association with Material Design.
+We have seen too many people confusing Material-UI with a Google effort or a synonym of Material Design.
+
+We are now called **MUI**, it stands for **M**aterials to Build **UI**s.
+and is pronounced [/ɛmjuːaɪ/](http://ipa-reader.xyz/?text=%C9%9Bmju%CB%90a%C9%AA).
+
+It's shorter, it distinguishes ourselves, it's already how many people abbreviate us.
+More importantly, it prepares the release of unstyled/headless components, a second design system to complement Material Design, and more.
+
+### A new domain name
+
+A new name wouldn't resonate without a new domain name.
+We have moved home from https://material-ui.com/ to https://mui.com/, an 3 domain laters domain.
```

Comment:
> an 3 domain laters domain

I don't get this part

Reply:
Fixed, with Danilo's suggestion. A 3-letter words that is faster to type.